doc/developers/contributing.rst

```diff
@@ -1172,3 +1164,4 @@ that implement common linear model patterns.
 
 The :mod:`sklearn.utils.multiclass` module contains useful functions
 for working with multiclass and multilabel problems.
+
```

Comment:
I am confused where did the other changes in the file come from? I did not make those changes.